tests/Support/Traits/CssDataProviders.php

```diff
@@ -29,19 +29,19 @@ public function provideEquivalentCompleteCss(): array
     {
         $datasetsWithAtMediaRuleSelectorListAndPropertyDeclaration = self::crossDatasetWithItself([
             'unminified CSS with `@media` rule, selector list, and property declaration'
-                => ['@media screen { html, body { color: green; } }'],
+            => ['@media screen { html, body { color: green; } }'],
```

Comment:
I think these should all still be indented, for readability.  We indent the second line of assignment statements, e.g.
```php
$someLongVariableName
    = $this->someLongMethodName($someLongListOfArgumentsThatFitOnOneLine);
```
This is not much different; it's just an assignment of an array element within the array declaration.

Reply:
I have undone this change now.